app/src/main/java/com/metro/gwuexplorer/AlertAdapter.kt

```diff
@@ -9,28 +9,18 @@ import android.widget.TextView
 
 class AlertAdapter constructor(private val alerts: List<Alert>) : RecyclerView.Adapter<AlertAdapter.ViewHolder>() {
     override fun onCreateViewHolder(parent: ViewGroup, p1: Int): ViewHolder {
-        // Open & parse our XML file
         val itemView: View = LayoutInflater.from(parent.context).inflate(R.layout.row_alert, parent, false)
-
-        // Create a new ViewHolder
         return ViewHolder(itemView)
     }
-
     override fun getItemCount(): Int = alerts.size
-
-    override fun onBindViewHolder(p0: ViewHolder, p1: Int) {
-
-        val currentAlerts = alerts[p1]
-
-        p0.usernameTextView.text = currentAlerts.stationName
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val currentAlerts = alerts[position]
+        holder.lineNameTextView.text = currentAlerts.lineName
+        holder.alert_contentTextView.text = currentAlerts.description
```

Comment:
Should have consistent naming... the previous variable is using camel case and this variable is using underscores.